apps/codelab/src/app/codelabs/angular/material/material.module.ts

```diff
@@ -21,7 +21,7 @@ import { FormsModule } from '@angular/forms';
 const routes = RouterModule.forChild([
   {
     path: '',
-    redirectTo: '/material/intro',
+    redirectTo: '/angular/material/intro',
```

Comment:
What about every single other milestone in the angular folder?
Do the same?

Reply:
Done